Solutions/MimecastTIRegional/Data Connectors/MimecastTIRegional_API_AzureFunctionApp.json

```diff
@@ -0,0 +1,124 @@
+{
+  "id": "MimecastTIRegionalConnectorAzureFunctions",
+  "title": "Mimecast Intelligence for Microsoft - Azure Sentinel",
+  "publisher": "Mimecast",
+  "descriptionMarkdown": "The data connector for Mimecast Intelligence for Microsoft provides regional threat intelligence curated from Mimecast’s email inspection technologies with pre-created dashboards to allow analysts to view insight into email based threats, aid in incident correlation and reduce investigation response times.  \nMimecast products and features required: \n- Mimecast Secure Email Gateway \n- Mimecast Threat Intelligence\n",
+  "graphQueries": [],
+  "sampleQueries": [],
+  "dataTypes": [
+    {
+      "name": "Event(ThreatIntelligenceIndicator)"
+    }
+  ],
+  "connectivityCriterias": [],
```

Comment:
Connectivity criteria should not be empty.

Reply:
@NikTripathi Done.